pkg/util/system/util_test_tool.go

```diff
@@ -30,15 +30,11 @@ type FileTestUtil struct {
 	t *testing.T
 }
 
-// Creates a new test util for the specified subsystem
+// NewFileTestUtil creates a new test util for the specified subsystem
 func NewFileTestUtil(t *testing.T) *FileTestUtil {
-	tempDir, err := ioutil.TempDir("/tmp", "koordlet_test")
+	tempDir := t.TempDir()
```

Comment:
It seems `(*FileTestUtil).Cleanup()` is useless with this patch and its references are removed. 
Can we remove the `Cleanup()` since its work would be done implicitly?

Reply:
+1